pkg/devfile/adapters/common/utils.go

```diff
@@ -223,20 +232,15 @@ func IsPortPresent(endpoints []common.Endpoint, port int) bool {
 	return false
 }
 
-// IsRestartRequired returns if restart is required for devrun command
-func IsRestartRequired(command common.DevfileCommand) bool {
-	var restart = true
-	var err error
-	rs, ok := command.Exec.Attributes["restart"]
-	if ok {
-		restart, err = strconv.ParseBool(rs)
-		// Ignoring error here as restart is true for all error and default cases.
-		if err != nil {
-			klog.V(4).Info("Error converting restart attribute to bool")
-		}
-	}
+// GetCommandsMap returns a mapping of all of devfile command names to their corresponding DevfileCommand struct
+// Allowing us to easily retrieve the DevfileCommand of any command listed in a composite command
+func GetCommandsMap(commands []common.DevfileCommand) map[string]common.DevfileCommand {
```

Comment:
This is not used anywhere. I guess it was removed recently.

Reply:
ya, thanks it got added during rebase, fixed now